identity-tools/developer_identity/create-dev-identity.sh

```diff
@@ -0,0 +1,256 @@
+    -s <api_server_url>           api server address (default: 'https://api.us-east-1.mbedcloud.com')
+    -k <k8s_server_url>.          edge kubernetes server address (default: 'https://edge-k8s.us-east-1.mbedcloud.com')
+    -p <serial_number_prefix>     serial number prefix (default: 'DEV0')
+    -n <account_id>               account identifier (mandatory)
+    -o <output_directory>         output directory of identity.json (default: './')
+    -i <device_id>                edge-core's internal-id (mandatory)
+    -w <hw_type>                  hardware version of the gateway, refer configurations section in
+                                  $DEVID_CLI_DIR/radioProfile.template.json#L228 (default: 'rpi3bplus')
+    -r <radio_config>             radio configuration of the gateway, refer configurations section in
+                                  $DEVID_CLI_DIR/radioProfile.template.json#L228 (default: '00')
+    -l <led_config>               status led configuration of the gateway (default: '01')
+
+  Examples:
+
+    Typical usage, use the default values to generate an identity file. This will create random uuids for deviceID and accountID.
+
+        $cli_name -d
+
+    Generate a default identity file but with integration lab cloud addresses
+
+        $cli_name -d -m mbedcloudintegration
+
+    Generate a default identity file but relace the hardware version and radio configuration to specified value
+
+        $cli_name -d -w delledge3000 -r 01
+"
+}
+
+[ ! -n "$1" ] && cli_help && exit 1
+[ "$1" == "-v" ] && [ ! -n "$2" ] && cli_help && exit 1
+
+OPTIND=1
+
+while getopts 'hvVdm:c:g:s:k:p:n:o:i:w:r:l:' opt; do
+    case "$opt" in
+        h|-help)
+            cli_help
+            exit 0
+            ;;
+        v)
+            VERBOSE=1
+            ;;
+        V)
+            echo "$(cat $DEVID_CLI_DIR/VERSION)"
+            exit 0
+            ;;
+        d)
+            USE_DEFAULT=1
+            CLOUD_LAB="us-east-1.mbedcloud"
+            SERIAL_NUMBER_PREFIX="DEV0"
+            RADIO_CONFIG="00"
+            HW_VERSION="rpi3bplus"
+            LED_CONFIG="01"
+            OUTPUT_DIR="./"
+            LwM2M_URL="coaps://lwm2m.us-east-1.mbedcloud.com"
+            API_URL="https://api.us-east-1.mbedcloud.com"
+            GW_URL="https://gateways.us-east-1.mbedcloud.com"
+            k8s_URL="https://edge-k8s.us-east-1.mbedcloud.com"
+            ;;
+        m)
+            CLOUD_LAB="$OPTARG"
+            case "$CLOUD_LAB" in
+                mds-integration-lab|mbedcloudintegration)
+                    LwM2M_URL="coaps://mds-integration-lab.dev.mbed.com"
+                    API_URL="https://lab-api.mbedcloudintegration.net"
+                    GW_URL="https://gateways.mbedcloudintegration.net"
+                    k8s_URL="https://edge-k8s.mbedcloudintegration.net"
+                    ;;
+                mds-systemtest|mbedcloudstaging)
```

Comment:
Just curious. What's `mds`?

Reply:
I think it was the name used before Pelion Cloud. Mbed Device Service, I think it expands to.